FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -369,10 +319,11 @@ class ScreeningParseState(Enum):
         'Te zien na ': ScreenedFilmType.SCREENED_AFTER,
         'Gepresenteerd als onderdeel van ': ScreenedFilmType.PART_OF_COMBINATION_PROGRAM,
         'Wordt vertoond in combinatie met ': ScreenedFilmType.DIRECTLY_COMBINED}
+    re_num_screen = re.compile(r'^(?P<theater>.*?)\s+(?P<number>\d+)$')
```

Comment:
Please consider to move this regex to the corresponding method in the planner interface.

Reply:
Won 't do.
Will mess up the IDFA version.